Decompiler/PscDecompiler.cpp

```diff
@@ -61,10 +56,6 @@ Decompiler::PscDecompiler::PscDecompiler(const Pex::Function &function, const Pe
     m_Object(object),
     m_CommentAsm(commentAsm)
 {
-#ifdef TRACE_DECOMPILATION
```

Comment:
The removed parts of the trace logging should probably be added back in future commits, such as what was removed here.